zingolib/src/wallet.rs

```diff
@@ -199,13 +202,17 @@ pub enum WalletBase {
     Ufvk(String),
     /// Unified spending key
     Usk(Vec<u8>),
+    #[cfg(feature = "ledger-support")]
+    Ledger,
 }
+
 impl WalletBase {
     pub fn from_string(base: String) -> WalletBase {
-        if (&base[0..5]) == "uview" {
-            WalletBase::Ufvk(base)
-        } else {
-            WalletBase::MnemonicPhrase(base)
+        match base {
+            _ if &base[0..5] == "uview" => WalletBase::Ufvk(base.clone()),
+            #[cfg(feature = "ledger-support")]
+            _ if &base[0..5] == "ledger" => WalletBase::Ledger,
```

Comment:
0..6 (range is exclusive of end index)

Reply:
..= can be used to create an inclusive range, if 0..=5 would be clearer